cert-tools/snapstore/src/snapstore/client.py

```diff
@@ -0,0 +1,84 @@
+"""
+A client for interacting with endpoints of the snap Store API.
+
+Ref: https://api.snapcraft.io/docs/
+"""
+
+import requests
+import os
+from typing import Optional
+
+from snapstore.auth import AuthClient
+
+
+class SnapstoreClient:
+    """
+    Interact with endpoints of the snap Store API
+    """
+
+    snapstore_url = "https://api.snapcraft.io"
+
+    def __init__(self, token: Optional[str] = None):
+        self.token = token or os.environ.get("UBUNTU_STORE_AUTH")
+
+    def create_headers(
+        self,
+        store: Optional[str] = None,
+        headers: Optional[dict] = None
+    ) -> dict:
+        """
+        Return a dict containing the headers for all HTTP requests to the
+        snap Store API (combine "standard" headers request-specific ones).
+        """
+        if headers is None:
+            headers = {}
+        authorization = (
+            AuthClient.authorization_from_token(self.token)
+            if self.token else None
+        )
+        return {
+            "Snap-Device-Series": "16",
+            **headers,
```

Comment:
```suggestion
        authorization = (
            AuthClient.authorization_from_token(self.token)
            if self.token else None
        )
        return {
            "Snap-Device-Series": "16",
            **(headers if headers else {}),
```
just to follow the same style as below

Reply:
Done.